.github/CONTRIBUTING.md

```diff
@@ -92,14 +94,14 @@ Once you've found an issue you'd like to work on, please follow these steps to m
     * If the program file is a GUI application _and_ it doesn't accept any commandline arguments, no need to add it in `bin`.
     * If an array contains only one item, write it as a string.
     * If the application provides _only_ a 32bit download, the `architecture` field is not required. In all other cases, `architecture` field is mandatory.
-5. Test your manifest by installing, uninstalling, checking functionality, persistence etc.
-6. Confirm that manifest gets updated automatically - see [App Manifest Autoupdate](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate).
-7. Submit your pull request. Titles should follow these rules:
+4. Test your manifest by installing, uninstalling, checking functionality, persistence etc.
+5. Confirm that manifest gets updated automatically - see [App Manifest Autoupdate](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate).
+6. Submit your pull request. Titles should follow these rules:
     * If it's a new manifest, use `<app name>: Add version <version>`.
     * If it's an update to an existing manifest, use `<app name>@<version>: <small description>`.
     * If it's something related to maintenance, use `(chore): <small description>`.
-8. Add a comment starting with "/verify" after raising the PR - this will kick in the automatic manifest verifier.
-9. Wait for code review and address any issues raised as soon as you can.
+7. Add a comment starting with "/verify" after raising the PR - this will kick in the automatic manifest verifier.
```

Comment:
The '/verify' is not needed for PR opening, or the mechanism is broken? It should be triggered automatically IMO.

Reply:
😂 it does not seem to work like a hard-working robot, as you will see if you checked the latest PR in extras repo.

Reply:
> The '/verify' is not needed for PR opening, or the mechanism is broken? It should be triggered automatically IMO.

It seems like a security feature of GitHub. The verifier is run automatically only when a PR is raised by a maintainer. For outside contributors, the comment is needed.